index.html

```diff
@@ -64,6 +64,16 @@ <h2>Meet the Team</h2>
 
     <section id="contact">
         <h2>Contact</h2>
+        <div class="flex-row">
+            <div class="felx-col">
```

Comment:
Think there is a typo here with class name

Reply:
how weird, i already caught and changed that before I pushed but guess it must not have committed. thanks for the spot!